src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.System/Properties/Resources.resx

```diff
@@ -137,6 +148,10 @@
     <value>Lock</value>
     <comment>This should align to the action in Windows of a locking your computer.</comment>
   </data>
+  <data name="Microsoft_plugin_sys_lock_confirmation" xml:space="preserve">
+    <value>You are about to lock this computer, are you sure?</value>
```

Comment:
Maybe it should say: `You are about to lock this account, are you sure?`. I think it is a more precise statement. The same applies to some other confirmation messages. What do you think?

Reply:
I feel like you are locking the computer, not the account.  To me, if you lock the account, you'd have to have somebody unlock your account before you could log back in.